health.go

```diff
@@ -19,13 +19,13 @@ package marathon
 // HealthCheck is the definition for an application health check
 type HealthCheck struct {
 	Command                *Command `json:"command,omitempty"`
-	Protocol               string   `json:"protocol,omitempty"`
-	Path                   string   `json:"path,omitempty"`
-	GracePeriodSeconds     int      `json:"gracePeriodSeconds,omitempty"`
-	IntervalSeconds        int      `json:"intervalSeconds,omitempty"`
-	PortIndex              int      `json:"portIndex,omitempty"`
+	Protocol               string   `json:"protocol,"`
```

Comment:
oh, you are right: via the ui you can't leave these fields empty, but via api it fills in default values